### DISCUSSION TITLE
git_status symbols only explained in source?

### DISCUSSION THREAD

**Discussion by bazz1tv** [starship/starship#6027] [Q&A]
Hello, I'm completely new to starship, so please bare with me. It seems there is nowhere to learn the meaning of the git_status module's symbols outside of the [source code itself](https://github.com/starship/starship/blob/00d3dc86a21d11aede96f81ffbe49babe487984e/src/modules/git_status.rs#L18)

https://github.com/starship/starship/blob/00d3dc86a21d11aede96f81ffbe49babe487984e/src/modules/git_status.rs#L18-L29

I'm familiar with `starship explain`, but that provided nothing on the meaning of the symbols. Am I missing something, perhaps a manual page, or is this the current state of affairs?

```
❯ starship explain

 Here's a breakdown of your prompt:
 "git " (<1ms)               -  The current working directory
 "on  master " (<1ms)       -  The active branch of the repo in your current directory
 "[\$?] " (1ms)              -  Symbol representing the state of the repo
 "via  v11.4.0-gcc " (2ms)  -  Your C compiler type
 "❯ " (<1ms)                 -  A character (usually an arrow) beside where the text is entered in your terminal
```

Thanks,
bazz

**Comment by davidkna** [comment]
The default symbol mapping is also explained [on the website](https://starship.rs/config/#git-status).
